sklearnex/tests/test_run_to_run_stability.py

```diff
@@ -73,23 +73,41 @@ def eval_method(X, y, est, method):
     est.fit(X, y)
 
     if method:
-        if method != "score":
-            res = getattr(est, method)(X)
+        attr = getattr(est, method)
+        if method == "inverse_transform":
+            # PCA's inverse_transform takes (n_samples, n_components)
+            data = (
+                (X[:, : est.n_components_],) if X.shape[1] != est.n_components_ else (X,)
+            )
+        elif method not in ["score", "partial_fit", "path"]:
+            data = (X,)
         else:
-            res = est.score(X, y)
+            data = (X, y)

```

Comment:
This code is duplicated here and in `sklearnex/tests/test_patching.py`.
Is it possible to have it in some common place?

Reply:
Good idea, will change that.